src/faebryk/libs/picker/picker.py

```diff
@@ -85,7 +95,7 @@ def pick_module_by_params(module: Module, options: Iterable[PickerOption]):
     if option.pinmap:
         module.add_trait(can_attach_to_footprint_via_pinmap(option.pinmap))
 
-    option.part.supplier.attach(module, option.part)
+    option.part.supplier.attach(module, option.part, option.partid)
```

Comment:
keep as it was

Reply:
changed it to pass option instead of part.option